docs/openapi/components/schemas/common/SIMASteelImportProductSpecifier.yml

```diff
@@ -24,13 +24,6 @@ properties:
     $linkedData:
       term: steelProduct
       '@id': https://w3id.org/traceability#SteelProduct
-  productCategory: 
```

Comment:
We can't remove this, they are based on SIMA's documentation.

Reply:
that means it should be in the example, right?

Reply:
Yep!

Reply:
@nissimsan what would the example value look like for this?

Reply:
I think I figured this out went with 'Long (Carbon and Alloy)' based off of https://www.trade.gov/steel-products-hts-codes